build/gengo.bzl

```diff
@@ -0,0 +1,171 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Bazel macros for using the k8s.io/gengo framework to generate zz_generated.FOO.go files.
+#
+# Usage:
+#   1) Create a rule that builds all the generated files, in //build for example
```

Comment:
might be worth documenting why you need a `k8s_gengo_all` rule.

Reply:
Done